assets/crd/authorizationv1-local-apiservice.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
```

Comment:
These are local API services, meaning that the aggregation layer should send requests for these groups to the local kube-apiserver (where the CR handler will serve them). Inside the kube-apiserver process, there's a CRD registration controller that is responsible for watching CRDs and automatically creating local APIServices as needed. On OpenShift, this is explicitly disabled for SCC and RBR because of these patches:

- https://github.com/openshift/microshift/blob/26d1d58423f8c4a8e7f6465f424cfb88c99e70b8/vendor/k8s.io/kubernetes/pkg/controlplane/controller/crdregistration/crdregistration_controller.go#L197-L203
- https://github.com/openshift/microshift/blob/26d1d58423f8c4a8e7f6465f424cfb88c99e70b8/vendor/k8s.io/kube-aggregator/pkg/apiserver/patch_always_local_delegate.go#L37-L49
- https://github.com/openshift/microshift/blob/26d1d58423f8c4a8e7f6465f424cfb88c99e70b8/vendor/k8s.io/kubernetes/openshift-kube-apiserver/enablement/intialization.go#L59-L62

So this PR is doing what the CRD registration controller normally does when it sees a CRD.

Reply:
And the reason we don't need to configure any authentication or transport fields because these APIServices are handled in-process.

Reply:
ok, this explains why I see API services created from other CRDs like topolvm, route, etc.